config/identify_system.go

```diff
@@ -6,12 +6,14 @@ import (
 )
 
 // Figure out if we're self-hosted or on RDS, as well as what ID we can use - Heroku is treated separately
-func identifySystem(config ServerConfig) (systemType string, systemScope string, systemScopeFallback string, systemID string) {
+func identifySystem(config ServerConfig) (systemID string, systemType string, systemScope string, systemIDFallback string, systemTypeFallback string, systemScopeFallback string) {
```

Comment:
This is kind of a lot of return values, but I don't have any great ideas here. We could use a struct, but given this is only called in two places, and the values are immediately assigned to fields of a Config object, I don't know if that's helpful.

Reply:
I think that's fine for now - a struct seems like the way to go if we wanted to improve on it (possible a struct for the set of id+type+scope, so its two return values?)